e2e/scripts/st_nested_columns_3.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+import streamlit as st
+
+st.set_page_config(layout="wide", initial_sidebar_state="collapsed")
+
+"""
+# Example 3
+https://discuss.streamlit.io/t/support-for-nested-columns/28810
+
+This is a super complex layout. It requires 2 levels of nesting and I even left out
+a few details of the original request. We shouldn't support this!
+
+---
+"""
+np.random.seed(0)
+left_banner, main_area = st.columns([0.05, 0.95])
+with left_banner:
+    for emoji in ["⚒️", "❌", "♻️", "🗑️", "🔌", ...]:
```

Comment:
nit: I abbreviated some of the emoji in an earlier comment by writing `...` (which actually works because it's the same thing as writing `pass` here), but I think that may have been copied here